index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Home Page</title>
+</head>
+<body>
+  <div class="grid-container">
+    <nav class="navbar">
+      <div class="hamburger small-screen">
+        <span class="hamburger-bar"></span>
+        <span class="hamburger-bar"></span>
+        <span class="hamburger-bar"></span>
+      </div>  
+      <a href="#" class="brand"><img src="images/logo.png" alt="logo"  class="brand">Nyegenyege</a>
+      <ul class="nav-list flex-row">
+        <li class="nav-item "><a href="#" class="active">About</a></li>
+        <li class="nav-item"><a href="#">program</a></li>
+        <li class="nav-item"><a href="#">Join</a></li>
+        <li class="nav-item"><a href="#">Sponsor</a></li>
+        <li class="nav-item"><a href="#">News</a></li>
+        <li class="nav-item " ><a href="#">Campaign</a></li>
+      </ul>
+    </nav>
+    <div class="home">
+      <header class="title">
+        <p class="tagline spacing">"MTN Nyegenyege welcomes all"</p>
+        <h2 class="page-title page-title-1 spacing">THE MTN NYEGENYEGE FESTIVAL 2022</h2>
+        <p class="introduction spacing">
+          Nyege Nyege is a collective in Kampala, Uganda that 
+          promotes outsider music, primarily electronic, by African artists. 
+          It was founded in 2013 by expats Arlen Dilsizian and Derek Debru. 
+          In addition to organizing ongoing parties and an artist residency
+          the collective runs.
+        </p>
+        <div class="event-date spacing">2022.8.12 (MON) ~ 18(SUN)</div>
+        <p class="venue spacing">The event will be held in Jinja at the Nile</p>
+      </header>
+      <section class="main-program">
+        <h2 class="program-title">main program</h2>
+        <span class="guide-bar"></span>
+        <div class="page-section activities">
+          <div class="lecture program-panel">
+            <div class="programe-icon">
+              <img src="images/program_icon_01.png" alt="">
+              <p class="icon-title">Lecture</p>
+            </div>
+            <p class="description">
+              Debru was born in Burundi but grew up in Belgium, 
+              moving to Uganda in 2010.
+            </p>
+          </div>
+          <div class="lecture program-panel">
+            <div class="programe-icon">
+              <img src="images/program_icon_02.png" alt="">
+              <p class="icon-title">CC Exhibtion</p>
```

Comment:
Please ensure that the pages look almost identical to the original design. Please use the same `color, fonts` and also implement the `hover effect` on the cards as in the original design. Please ensure that the pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.

* Basic color swatch: #272A31, #EC5242, #D3D3D3

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/163654962-bf5175a5-ff41-4bb4-bf8d-5bfd3d19e6a8.png) | ![image](https://user-images.githubusercontent.com/10332499/163654939-ccd92410-6192-43b5-85f8-0cbcfd719ff4.png)